nifi-nar-bundles/nifi-extension-utils/nifi-event-transport/src/main/java/org/apache/nifi/event/transport/netty/channel/StandardChannelInitializer.java

```diff
@@ -37,6 +39,8 @@
 
     private Duration writeTimeout = Duration.ofSeconds(30);
 
+    private Duration idleTimeout = Duration.ofSeconds(30);
```

Comment:
As in the `NettyEventSenderFactory`, it seems like the internal default should be `0`.